lib/puppet/network/http.rb

```diff
@@ -5,6 +5,9 @@ module Puppet::Network::HTTP
   MASTER_URL_PREFIX = "/puppet"
   CA_URL_PREFIX = "/puppet-ca"
 
+  PUPPET_API_VERSIONS = "v3, v2"
```

Comment:
I could be behind the times but I thought puppet 4 was `v3`only for the `puppet` part of the API?